### ISSUE TITLE
Custom Injected CSS & Universal Dark Mode via Dark Reader at the same time.

### ISSUE THREAD

**Issue by ccioanca** [getferdi/ferdi#346]
Currently, it doesn't seem that you can have both darkmode.css and universal dark mode working in sync. This would be a very helpful option. 

If for some reason Dark Reader messes something up, it would allow the user to inject their own custom css via the darkmode.css option to fix said mess-up. This would also allow the user to make any edits they might want to make, regardless of reason. I, for example, like to give fb messenger and discord collapsing side bars so that I have more space for the message/content area on a vertical monitor. It's a special use case for sure, but allowing this feature would be a catch-all for all special and not-so-special use cases. 

I understand that this might slightly change the function of the darkmode.css feature. I suggest we allow another specially named file for css that can work in tandem with universal dark mode, because being able to choose one or the other (darkmode.css vs Universal Dark Mode via Dark Reader) is ALSO a feature that shouldn't be removed. 

I propose we keep the current functionality of darkmode.css and Dark Reader, but add the option of a "userstyles.css" as well. 

This might be too closely related to [Issue 83](https://github.com/getferdi/ferdi/issues/83) issue. If so, please merge the two issues. Or delete this and I will post this as a comment there.

**Comment by fsevenm** [comment]
I like the idea, why not make an option to allow user style works along with Dark Reader. In my case, Universal dark that is handled by Dark Reader does not make everything dark, some little things still with its default color which are really weird for me. 

WhatsApp for example, with Universal Dark mode activated, it will look like this
![2020-02-10_15-59](https://user-images.githubusercontent.com/29963765/74131248-e6046900-4c1e-11ea-92ae-a5d4ed3c9373.png)

Small white triangles beside chat balloons look weird, green message-out balloons and the default WA background look weird too, I guess.

What I did to fix that, inside `/home/<username>/.config/Ferdi/recipes/whatsapp/`, I made a CSS file called `userstyles.css`, inside the file, I overrode some styles that caused the issues. Then, I edited `webview.js` also in the same folder, I added this line
```js
Franz.injectCSS(path.join(__dirname, 'userstyles.css'));
```
after `Franz.injectCSS(path.join(__dirname, 'service.css'));`, saved the files, reloaded the service then boom, the view transformed to
![2020-02-10_16-02](https://user-images.githubusercontent.com/29963765/74132112-d423c580-4c20-11ea-8630-2e81d7b318a6.png)

That's all I did to treat weird stuff caused by Universal Dark mode.

I hope this feature is added to the next release of Ferdi.
